src/Core/Domain/ShowcaseCard/ConfigurationMap.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShop\PrestaShop\Core\Domain\ShowcaseCard;
+
+use PrestaShop\PrestaShop\Core\Domain\ShowcaseCard\ValueObject\ShowcaseCard;
+
+/**
+ * Maps showcase card names to configuration names from ps_configuration
+ */
+class ConfigurationMap
+{
+    /**
+     * Template used to create configuration names for "closed status"
+     */
+    const CLOSED_TEMPLATE = 'PS_SHOWCASECARD_%s_CLOSED';
+
+    /**
+     * @var array
+     */
+    private $closedCardConfiguration;
+
+    public function __construct()
+    {
+        $this->closedCardConfiguration = [
+            ShowcaseCard::SEO_URLS_CARD => sprintf(self::CLOSED_TEMPLATE, 'SEO'),
```

Comment:
i know it's not a big deal, but how about `SEO_URLS` in `sprintf` as well? just for consistency with `ShowcaseCard::SEO_URLS_CARD`

Reply:
You mean this?

```diff
-ShowcaseCard::SEO_URLS_CARD => sprintf(self::CLOSED_TEMPLATE, 'SEO'),
+ShowcaseCard::SEO_URLS_CARD => sprintf(self::CLOSED_TEMPLATE, 'SEO_URLS'),